README.md

```diff
@@ -165,12 +165,12 @@ Example in VSCode:
 
 ## Contributing, Credits, etc
 
-Author: [@hmil](https://github.com/hmil)  
```

Comment:
Note: The double space at the end of line is important to mark a line break in markdown. I'll re-add it in the next commit